components/ws-daemon/pkg/content/initializer.go

```diff
@@ -390,13 +389,28 @@ func (rs *remoteContentStorage) Download(ctx context.Context, destination string
 	}
 
 	span.SetTag("URL", info.URL)
-	resp, err := http.Get(info.URL)
+
+	tempFile, err := os.CreateTemp("", "remote-content-")
+	if err != nil {
+		return true, xerrors.Errorf("cannot create temporal file: %w", err)
+	}
+	defer tempFile.Close()
+	defer os.Remove(tempFile.Name())
+
+	args := []string{
+		"-x16", "-j12",
```

Comment:
Apparently, by default, retries are repeated 5 times. This seems a bit much.
https://aria2.github.io/manual/en/html/aria2c.html#cmdoption-m